### ISSUE TITLE
Current version of Nivo breaks when server side rendering

### ISSUE THREAD

**Issue by citypaul** [plouc/nivo#1975]
**Describe/explain the bug**
Hi there!

Firstly I just wanted to thank you for this awesome library, as I'm sure you're saving lots of people a huge amount of time and effort.

I have found there is an issue at the moment when trying to render charts with SSR with the current version of Nivo. I have also seen that an issue was already created for this and a fix was created, but it appears that this fix has not yet made it into a release, so I thought I'd create this issue for two reasons - firstly to remind the maintainers of this library that this is still an issue until the fix has been released, and secondly to help anybody else who is currently experiencing the same issue.

The Github issue that referenced this bug is here:  https://github.com/plouc/nivo/issues/1889

As stated in the issue itself, a fix for this was created here: https://github.com/plouc/nivo/commit/2763c4a28e5376457823b3be7940745c7f6feb8c

Just to make the issue I'm creating now easier to Google, the error that is logged is:

```
ReferenceError: ResizeObserver is not defined
```

For people who find this issue before the fix has made it to a release, you can temporarily fix by locking down your nivo dependencies to a previous version.

For me, locking down like this worked:

```sh
    "@nivo/core": "0.78.0",
    "@nivo/line": "0.78.0",
```

Again, thanks for all your great work on this library, and I'm only creating this issue to remind you that the issue currently remains and to help anybody who is also having the same problem at this time.

**Comment by pragmaticpat** [comment]
Confirming also that pinning to `0.78.0` is the solution for me as well. Looking forward to the proper fix, and happy to help with testing where needed.

**Comment by adamstickley** [comment]
Leaving `@nivo/radar` on `0.79.1` doesn't cause any issues for me, so can confirm pinning just `@nivo/core` back to `0.78.0` does the trick for my case as well.

**Comment by plouc** [comment]
It has been fixed in `0.80.0`.
